dev-docs/source/ReleaseChecklist.rst

```diff
@@ -110,15 +110,22 @@ with the Release Manager.
 Monday, 3 weeks
 ---------------
 
-*  Ensure that Manual testing begins. The instructions for Manual testing are found
-   `here <https://www.mantidproject.org/Unscripted_Manual_Testing>`__.
+*  Ensure that Manual testing begins. An overview of the Manual testing to be done is
+   found `here <Testing>`. Generate the Manual testing issues by following the instructions
```

Comment:
Rob, my apologies, this should be 
```rst
:ref:`here <Testing>`
```
I forgot the `:ref:` to treat it as an internal reference.

Reply:
Oops, I should have known!